conda_pack/core.py

```diff
@@ -705,10 +705,14 @@ def load_environment(prefix, on_missing_cache='warn'):
     managed = set()
     uncached = []
     missing_files = []
+    conda_in_env = False
     for path in os.listdir(conda_meta):
         if path.endswith('.json'):
             with open(os.path.join(conda_meta, path)) as fil:
                 info = json.load(fil)
+            if info['name'] == 'conda':
+                conda_in_env = True
```

Comment:
👍 A concise and obvious way to conditionally add the activate/deactivate scripts.